packages/authentication/test/unit/bindings.test.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from 'testlab';
+import * as http from 'http';
+import {Strategy} from '../..';
+import {getAuthenticatedUser} from '../..';
+import {supertest} from 'testlab';
+
+describe('Authentication Bindings', () => {
+  let strategy: Strategy;
+  let app: http.Server;
+  const mockUser: User = {id: 'mock-user', role: 'mock-role'};
+  interface User {
+    id: string;
+    role: string;
+  }
+
+  describe('getAuthenticatedUser()', () => {
```

Comment:
I would give a word of caution to using the name of the method as the name of the test, up until a year ago we used to do the same and it didn't really follow proper bdd and tightly coupled the test to the function, your describe should describe the expected behaviour not knowing how it achieved that behaviour.

Another reason we discourage it is that if the function name changes for whatever reason you now have to update the test and the describe and it's likely to be forgotten at some stage.